LondonBritishLibrary/orient/BLorient8823.xml

```diff
@@ -0,0 +1,266 @@
+                                        <width>215</width>   
+                                    </dimensions> 
+                                    <note><locus target="74"/> is only 120mm wide and contains one column on the recto. A small piece of parchment
+                                        is inserted after <locus target="#32"/> and is referred to in this description as f. 32bis, containing a pen trial and a 
+                                        Beta Israel eulogistic formula.</note>
+                                </extent>
+                                <foliation>Numbered 2-103, f. 1 missing. Quire numbers on three quires on <locus from="22" to="35"/>.
+                                        </foliation>
+                                <collation>
+                                    <note>According to <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">5</citedRange>
+                                        </bibl><locus from="22" to="35"/> composed of three numbered quires.</note>
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="42 43">
+                                    <locus from="1r" to="74r"/>
+                                </layout>
+                                <layout columns="2" writtenLines="23">
+                                    <locus from="75r" to="103r"/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <locus from="3r" to="44v"/>
+                                <desc>Fine regular script.</desc>
+                                <date notBefore="1700" notAfter="1799" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                            <handNote script="Ethiopic" xml:id="h2">
+                                <locus from="45r" to="103r"/>
+                                <desc>Good hand, large characters.</desc>
+                                <date notBefore="1700" notAfter="1799" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                            <handNote xml:id="h3" script="Ethiopic">
+                                <locus from="71ra" to="103ra"/>
+                                <desc>Good hand, large characters, especially from <locus from="95r" to="103r"/>.</desc>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="other">
+                                <locus target="#14 #29 #40 #63 #67 #98"/>
+                                <desc>The different parts of the manuscript are marked by small pieces of red yarn sewn onto the margins of the 
```

Comment:
These are probably leaf string markers. They can be described in the description of the binding
https://betamasaheft.eu/Guidelines/?id=bindingDescription